apps/cyberstorm-remix/app/p/components/ReportPackage/ReportPackage.tsx

```diff
@@ -147,22 +168,91 @@ export function ReportPackageForm(props: ReportPackageFormFullProps) {
             rootClasses="report-package__textarea"
           />
         </div>
-        {error && (
+        {strongForm.inputErrors ||
+        strongForm.refineError ||
+        strongForm.submitError ? (
           <div className="report-package__block">
-            <NewAlert csVariant="danger">{error}</NewAlert>
+            {strongForm.inputErrors ? (
+              <NewAlert csVariant="danger">
```

Comment:
Input specific errors should be shown next to the relevant input if at all possible.

Reply:
Not possible currently, as that would need mapping of fields to errors and anchoring to elements on the form. Additionally there has not been a design decision made on how errors should be displayed, for anything, so by that the errors should not be shown there.